personal-growth/investment-time.html.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Investment Time
+meta_description: >
+  At Nebulab, 10% of the time is dedicated to personal and company projects. Sounds interesting? Read on!
+---
+
+We use 10% of our time for personal growth. During this time, named "Investment Time", we do not work for clients.
+Instead, we focus on initiatives that improve our skills or contribute to the community.
+Here are some ideas for Investment Time:
+
+- Write a post on your personal blog or our [own](working-on-nebulab/blog)
+- Contribute to an open source project
+- Study a new tool or technology
+- Mentor a teammate and teach them a skill
+- Studying a language
+
+This is "Investment Time", because it's time you can invest in yourself to grow as a professional and progress
+in your career. It is the right time to work on personal open source projects (if you have any) or even relaxing
+a little bit if you feel stressed over doing too much client work.
+
+If you come up with an idea and you're not sure whether it's a good candidate for Investment Time, your
+mentor will be able to help you.
+
+## Tracking your Investment Time budget
+
+Every year your Investment Time budget will be 165 hours. You are required to track against this budget by
+using Harvest, there is a task named "Investment Time" under client Nebulab.
+
+Whenever you're doing something for your personal growth, whatever the activity might be, you should be tracking
+against that budget. There are a few exceptions: conferences, All Hands meetings, 1:1s and team retreats are not
+considered Investment Time and should not be tracked. When in doubt, talk to your mentor to figure out if a particular
+activity is Investment Time or something else.
+
+Once you hit the yearly budget you're done for that year and need to wait for the next one to use more Investment
+Time. Considering the full budget is given for every full year, if you joined Nebulab mid-year you might get a
```

Comment:
I'd avoid saying `might get` since you're going to get a prorated amount of the yearly budget - e.g., mid-year -> 82.5.